### ISSUE TITLE
Cluster Problem

### ISSUE THREAD

**Issue by nrmatsch** [azixus/ARK_Ascended_Docker#42]
Hey,
when i trying to create a cluster server, in the docker-compose.yml I have entered the following:

volumes:
        clusters: /var/www/clients/client1/web1/web/server/asa/cluster/

The following error message appears at ./manager.sh start:

validating /var/www/clients/client1/web1/web/server/asa/01/ARK_Ascended_Docker/docker-compose.yml: volumes.clusters must be a mapping or null

The cluster directory exists.
Thanks for any help

**Comment by Emruuu** [comment]
volumes:
- /var/www/clients/client1/web1/web/server/asa/cluster:/opt/shared-cluster

Try that. U forgot the second part.

**Comment by simmonsj17** [comment]
I'm new to docker and hosting a server, I've got 3 maps up but can't transfer between. It takes me to the map but character data doesn't transfer. I assume I don't have the volume right in docker-compose.yml file. This is my volume: - /opt/clusters:/opt/shared-cluster
Sometimes im dumb lmao but, this is my file path to 1 instance of the 3maps, /home/josh/tek-ark-cluster/ark_instance_1
Should I make a dir in a specific folder like in my tek-ark-cluster folder for it?
Then how would I set the volume in the docker-compose.yml file? Like this: -  /home/josh/tek-ark-cluster/shared-cluser:opt/shared-cluster?
Ty in advance

**Comment by Luk019** [comment]
You need to create a folder outside your Docker, for example, in the user folder or the working directory of the machine, e.g., /opt/yourfolder. Then, in the docker-compose.yml file, add the following line under volumes:

`- yourpathtofolder:/opt/shared-cluster`

In your example u write it good but u missed "/" beffore opt:

`- /home/josh/tek-ark-cluster/shared-cluster:/opt/shared-cluster`

**Comment by simmonsj17** [comment]
Thanks!! I was wondering if you may know why my game.ini file is erased / overwritten to blank upon startup?

**Comment by Luk019** [comment]
Yeah. U need to stop the server and then make a change in game.ini. If server is on he rewrite it all time with the old one.

PS. If it your first run he create the files. So first run it then stop it and change it:P

**Comment by simmonsj17** [comment]
Id suggest adding this to the docker-compose.yml file incase you want to setup pve start and stop times so the servers are all on the same times.
environment:
      - TZ=(your time zone) 
ex:
environment:
      - TZ=America/New_York

**Comment by Evocomms** [comment]
please can i have some help with this i have made a cluster,  cluster maps are visible but when i transfer i get the make character menu.

i have set my mounts as follows:

    volumes:
      - ./ark_data:/opt/arkserver
      - ./ark_backup:/var/backups/asa-server
      - /mnt/2TBSSD/Arkservers/clusters:/opt/shared-cluster
      - steam_data:/opt/steamcmd

i created the folder in /mnt/2TBSSD/Arkservers/clusters and even chmod 777 to the whole of Arkservers, it just seems that the shared-cluster does not mount.

any help would be greatly appreciated, thanks.

**Comment by simmonsj17** [comment]
> please can i have some help with this i have made a cluster, cluster maps are visible but when i transfer i get the make character menu.
> 
> i have set my mounts as follows:
> 
> ```
> volumes:
>   - ./ark_data:/opt/arkserver
>   - ./ark_backup:/var/backups/asa-server
>   - /mnt/2TBSSD/Arkservers/clusters:/opt/shared-cluster
>   - steam_data:/opt/steamcmd
> ```
> 
> i created the folder in /mnt/2TBSSD/Arkservers/clusters and even chmod 777 to the whole of Arkservers, it just seems that the shared-cluster does not mount.
> 
> any help would be greatly appreciated, thanks.

Make sure in the env file in extra dash options you need these  -clusterID=thisisarandomid -ClusterDirOverride="/opt/shared-cluster"

**Comment by Evocomms** [comment]
Thank you so much mate as i reread the readme i realised there was the ClusterDirOverride="/opt/shared-cluster missing and you have also confirmed it,

Thanks for the speedy reply mate i much appreciate it.
